content/en/docs/concepts/workloads/controllers/_index.md

```diff
@@ -13,7 +13,7 @@ Ultimately, your applications run as containers inside
 Pods would be a lot of effort. For example, if a Pod fails, you probably want to
 run a new Pod to replace it. Kubernetes can do that for you.
 
-You use the Kubernetes API to create workload
+You use the Kubernetes API to create a workload
 {{< glossary_tooltip text="object" term_id="object" >}} that represent a higher level
```

Comment:
and maybe this:
```suggestion
{{< glossary_tooltip text="object" term_id="object" >}} that represents a higher level
```
?

Reply:
Taken that suggestion on board and added a new commit. Also slightly reworded so it flows better (imo).